README.md

```diff
@@ -1 +1,7 @@
 # code_challenges
+## Code Wars
+---
```

Comment:
The 3 dashes `---` are redundant, Headings already include a linebreak. See this file here https://github.com/aepaysinger/code-challenges/blob/61aae568b0bc200ef7cab6abbb3e0e85adfa9e8d/README.md